tests/status.py

```diff
@@ -0,0 +1,23 @@
+import os
+import unittest
+
+import requests
+
+BASE_URL = os.environ.get('BASE_URL') or 'http://pay.dev'
+EXAMPLE_URL = BASE_URL
+UI_URL = BASE_URL + ':8000'
+SERVICE_URL = UI_URL + '/api/'
+
+
+class TestStatuses(unittest.TestCase):
+
+    def test_service_status(self):
+        res = requests.get(SERVICE_URL)
+        res.raise_for_status()
```

Comment:
it would be pretty straight forward to check this json output (perhaps in another test) to see if solitude is connected or not: http://payments-service.readthedocs.org/en/latest/topics/api.html#status